HomebrewFormula/instrumentald.rb

```diff
@@ -0,0 +1,25 @@
+class Instrumentald < Formula
+  head "https://github.com/Instrumental/instrumentald.git"
+  desc "A server agent that provides system monitoring and service monitoring. It's fast, reliable, runs on anything *nix, is simple to configure and deploy, and has a small memory footprint."
+  homepage "https://github.com/Instrumental/instrumentald"
+  url "https://github.com/Instrumental/instrumentald/releases/download/0.0.5/instrumentald_0.0.5_osx.tar.gz"
+  version "0.0.5"
+  sha256 "1c6eb9516c9a99d8ba55d1ab3a99d744c442f1aafbe2ded220302b76288c6023"
```

Comment:
Correct. I agree we should document this. I do not believe there to be any packaging instructions right now, My plan was to do a new PR for that.